shopping_app/lib/screens/product_details.dart

```diff
@@ -0,0 +1,191 @@
+                            child: Text(
+                              singleProduct.description,
+                              style: TextStyle(fontSize: 17),
+                            ),
+                          ),
+                        ],
+                      ),
+                    ),
+                    Padding(
+                      padding: EdgeInsets.only(top: 15, left: 20),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.start,
+                        children: [
+                          Text(
+                            "${singleProduct.price}\$",
+                            style: TextStyle(
+                                fontSize: 25, fontWeight: FontWeight.bold),
+                          ),
+                          Padding(
+                            padding: const EdgeInsets.only(left: 8.0, top: 8),
+                            child: Text(
+                              "1000\$",
+                              style: TextStyle(
+                                  fontSize: 15,
+                                  fontWeight: FontWeight.bold,
+                                  color: Colors.grey,
+                                  decoration: TextDecoration.lineThrough),
+                            ),
+                          ),
+                        ],
+                      ),
+                    ),
+                    Padding(
+                      padding: EdgeInsets.only(top: 15, left: 20),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.start,
+                        children: [
+                          Chip(
+                            avatar: CircleAvatar(
+                              backgroundColor: Colors.grey.shade800,
+                            ),
+                            label: Text(singleProduct.category[0]),
+                          ),
+                          Chip(
+                            avatar: CircleAvatar(
+                              backgroundColor: Colors.grey.shade800,
+                            ),
+                            label: Text(singleProduct.category[1]),
+                          )
+                        ],
+                      ),
+                    ),
+                    Align(
+                      alignment: Alignment.bottomCenter,
+                      child: Padding(
+                        padding: const EdgeInsets.only(top: 20.0),
+                        child: Row(
+                          children: [
+                            Container(
+                              height: 60,
+                              color: Colors.yellow,
+                              child: FlatButton(
+                                onPressed: () {},
+                                child: Row(
+                                  mainAxisAlignment: MainAxisAlignment.center,
+                                  children: [
```

Comment:
Here, we need to create a separate reusable widget

Reply:
Created Where Required